src/services/expenses.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const validator = require('./expenses.validator');
+
+let expenses = [];
+
+const initiate = (initialValue) => {
+  expenses = initialValue;
+};
+
+const getNewId = () => (
```

Comment:
It is possible to create one universal function for users and exponses for creating id